software/runtime/arch.ld.c

```diff
@@ -21,7 +21,7 @@ SECTIONS {
 
   // Stack size
   __stack_start = __l1_start;
-  __stack_end = __l1_start + (NUM_CORES * 0x400);
+  __stack_end = __l1_start + (NUM_CORES * 0x200);
```

Comment:
This sounds a bit dangerous and might break applications that need the stack size. Why do we need this for Terapool?